stdlib/3/gzip.pyi

```diff
@@ -40,7 +40,7 @@ class GzipFile(_compression.BaseStream):
     def writable(self) -> bool: ...
     def seekable(self) -> bool: ...
     def seek(self, offset: int, whence: int = ...) -> int: ...
-    def readline(self, size: int = ...) -> bytes: ...
+    def readline(self, size: Optional[int] = ...) -> bytes: ...
```

Comment:
This looks right; it gets passed through to `io.BufferedReader.readline` which accepts None.